test/integration-tests/test-server/schema/resolvers/book-resolvers-functions.ts

```diff
@@ -34,9 +36,15 @@ export const bookResolver = async (
     query: object,
     context: PolarisContext,
 ) => {
-    const { realityId } = context.headers;
+    const { realityId, includeLinkedOperation } = context.headers;
     if (!Number.isInteger(realityId as any)) {
         throw new UserInputError('please provide reality-id header');
+    } else if (includeLinkedOperation) {
+        const realityZeroContext: PolarisContext = getRealityZeroContext(context);
+        return BookModelPerReality(context)
```

Comment:
change to:
            const headers0 = Object.assign({}, context.headers);
            const context0 = Object.assign({}, context);
            headers0.realityId = 0;
            context0.headers = headers0;